elements/bulbs-video/components/revealed.js

```diff
@@ -247,7 +252,7 @@ export default class Revealed extends React.Component {
 
     player.setup(playerOptions);
 
-    GoogleAnalytics.init(player, videoMeta.gaPrefix);
+    GoogleAnalytics.init(player, videoMeta.sendAnalyticsEvent);
```

Comment:
passing in the function keeps the code a little DRYer, we're not repetitively passing in the prefix and 'event'